docs/styling.md

```diff
@@ -118,6 +118,27 @@ with the existing default template.
 > NOTE: There is no guarantee that your template will continue to work with future versions of F# Formatting.
 > If you do develop a good template please consider contributing it back to F# Formatting.
 
+## Customizing menu items by template
+
+You can add advanced stylings to the sidebar generated menu items by creating a new template for it.
+For adding a template via command line you can use the `--input` option or alternatively add them to the docs/ folder.
```

Comment:
This is slightly confusing. What you basically want to say is that the `fsdocs` will look for menu templates in the `--input` folders. This folder defaults to `docs`.
You are not really adding any template via the command line.

Also, make it clear that both templates need to exist before. Otherwise, it won't work.